htdocs/product/list.php

```diff
@@ -410,12 +412,12 @@
 if ($search_country)     $sql .= " AND p.fk_country = ".$search_country;
 if ($search_state)     $sql .= " AND p.fk_state = ".$search_state;
 if ($search_finished >= 0 && $search_finished !== '') $sql .= " AND p.finished = ".$search_finished;
-if ($search_accountancy_code_sell)        $sql .= natural_search('p.accountancy_code_sell', $search_accountancy_code_sell);
-if ($search_accountancy_code_sell_intra)  $sql .= natural_search('p.accountancy_code_sell_intra', $search_accountancy_code_sell_intra);
-if ($search_accountancy_code_sell_export) $sql .= natural_search('p.accountancy_code_sell_export', $search_accountancy_code_sell_export);
-if ($search_accountancy_code_buy)         $sql .= natural_search('p.accountancy_code_buy', $search_accountancy_code_buy);
-if ($search_accountancy_code_buy_intra)   $sql .= natural_search('p.accountancy_code_buy_intra', $search_accountancy_code_buy_intra);
-if ($search_accountancy_code_buy_export)  $sql .= natural_search('p.accountancy_code_buy_export', $search_accountancy_code_buy_export);
+if ($search_accountancy_code_sell)        $sql .= natural_search('p.accountancy_code_sell', clean_account($search_accountancy_code_sell));
```

Comment:
the clean_account method is defined into lib core/lib/accounting.lib.php
But the condition to include this file is not the same condition that the condition to have the call to clean_account being called. Surely a bug...